src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,398 @@
+you use the `load()` method of the audio element. This is one way for the
+browser to keep track of whether the user interacted with the element. Note
+that it may also help the playback to be smoother because the content will be
+loaded.
+
+<pre class="prettyprint">
+
+let audio = document.querySelector('audio');
+
+welcomeButton.addEventListener('pointerup', function(event) {
+  // User interacted with the page. Let's load audio...
+  <strong>audio.load()</strong>
+  .then(_ => { /* Show play button for instance... */ })
+  .catch(error => { console.log(error) });
+});
+
+// Later...
+playButton.addEventListener('pointerup', function(event) {
+  <strong>audio.play()</strong>
+  .then(_ => { /* Set up Media Session... */ })
+  .catch(error => { console.log(error) });
+});
+
+</pre>
+
+### Customize the notification
+
+When your web app is playing audio, you can already see a media notification
+sitting in the notification tray. On Android, Chrome does its best there to show
+appropriate information by using the document's title and the largest favicon
+it can find.
+
+<div class="clearfix"></div>
+<div class="attempt-left">
+  <figure>
+    <img src="/web/updates/images/2017/02/without-media-session.png" alt="Without Media Session">
+    <figcaption>With no Media Session</figcaption>
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="/web/updates/images/2017/02/with-media-session.png" alt="With Media Session">
+    <figcaption>With Media Session</figcaption>
+  </figure>
+</div>
+
+Let's see how to customize this media notification by setting some media
+session metadata such as the title, artist, album name, and artwork with the
+Media Session API.
+
+    if ('mediaSession' in navigator) {
+
+      navigator.mediaSession.metadata = new MediaMetadata({
+        title: 'Never Gonna Give You Up',
+        artist: 'Rick Astley',
+        album: 'Whenever You Need Somebody',
+        artwork: [
+          { src: 'https://dummyimage.com/96x96',   sizes: '96x96',   type: 'image/png' },
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
+        ]
+      });
+    }
+
+If your web app provides a playlist for instance, you may want to allow the
+user to navigate through your playlist directly from the media notification
+with some "Previous Track" and "Next Track" icons.
+
+    let audio = document.createElement('audio');
+
+    let playlist = ['audio1.mp3', 'audio2.mp3', 'audio3.mp3'];
+    let index = 0;
+    
+    navigator.mediaSession.setActionHandler('previoustrack', function() {
```

Comment:
Any idea why this is `setActionHandler` rather than `addEventListener`?

What if the user starts playing something without a playlist? Do I need to remove this handler somehow?

If I set the metadata to something else, do the handlers persist?